### PR TITLE
[HttpClient] minor fixes in RetryableHttpClient

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/HttpClient/RetryableHttpClient.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function __construct(HttpClientInterface $client, RetryDeciderInterface $
     public function request(string $method, string $url, array $options = []): ResponseInterface
     {
         if ($this->maxRetries <= 0) {
-            return $this->client->request($method, $url, $options);
+            return new AsyncResponse($this->client, $method, $url, $options);
```
[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Component/HttpClient/RetryableHttpClient.php:56]: this line was broken: the stream() method expects instances of AsyncResponse

[the remark this answers is not in the document]
Comment (stof) [reply src/Symfony/Component/HttpClient/RetryableHttpClient.php:56]: looks like some tests are missing then...

[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Component/HttpClient/RetryableHttpClient.php:56]: :grin: added

**Review by jderusse — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**